Makefile

```diff
@@ -21,5 +21,11 @@ test:
 ensure:
 	dep ensure
 
+darwin:
```

Comment:
this is unnecessary to run on an individual platform, but this is so CI can ensure the code still builds on both platforms

Reply:
Ah I see. Wouldn't we want to run + test on an actual mac, like we do in the tesseract CI?

Reply:
OK, yeah, I briefly considered that but then figured there wasn't any watcher_test.go so it didn't really matter yet (plus running cross-platform tests from a makefile is harder). That might have been silly, especially since it turns out notify_test tests watcher. Testing both in CI makes sense, thanks!